login/Models/RoleModel.js

```diff
@@ -0,0 +1,15 @@
+const sq =  require("../DataConnection.js");
```

Comment:
sq doesn't really tell a whole lot about the variable.

Reply:
Yea,i will rename it